<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DwarfDetailsDock</class>
 <widget class="QDockWidget" name="DwarfDetailsDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>564</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="styleSheet">
   <string/>
  </property>
  <property name="windowTitle">
   <string>Dwarf Details</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,2,10,0">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="lbl_dwarf_name">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_translated_name">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="indent">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_profession">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="indent">
         <number>6</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="gb_attributes">
      <property name="title">
       <string>Attributes</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Happiness</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="pb_happiness">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string>QProgressBar:horizontal {
	border: 1px solid grey;
	border-radius: 4px;
	background-color: 
		qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, 
			stop:0 rgba(255, 0, 0, 255), 
			stop:0.5 rgba(255,255,0,255),
			stop:1 rgba(0,255,0,255)
		);
	text-align: right;
	margin-right: 7ex;
}

QProgressBar::chunk {
	border-right: 3px solid rgba(0, 0, 0, 64)
	/*	background: rgba(255, 255, 255, 10);*/

}</string>
           </property>
           <property name="maximum">
            <number>150</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="format">
            <string>%v</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gb_skills">
      <property name="title">
       <string>Skills</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QScrollArea" name="sa_skills">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="sa_contents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>394</width>
            <height>299</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Dump Memory</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
